README.md

```diff
@@ -344,6 +344,9 @@ Thus, these should all be set on the JVM command line using `-D`.
   encryption/decryption operations would not require allocation and release of `EVP_CIPHER_CTX`
   structure. A common use case would be having long-running threads that each would get its
   own instance of `Cipher` class.
+* `com.amazon.corretto.crypto.provider.tmpdir`
```

Comment:
what happens if this points to a directory containing the libs we unpack into it? do they get overwritten or do the writes fail?

Reply:
We actually create a new directory here first and extract the files in there. Upon exit, the directory is marked to be deleted.